github-orgs/kubeflow/org.yaml

```
@@ -199,6 +199,7 @@ orgs:
         - ioandr
         - isinyaaa
         - IronPan
+        - IRONICBo
```

Please keep list alphabetical.
```suggestion
        - IRONICBo
        - IronPan
```

---

Oh, thanks, I have fixed it.